plugins/inventory/now.py

```diff
@@ -0,0 +1,443 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2021, XLAB Steampunk <steampunk@xlab.si>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+import os
+
+from ansible.errors import AnsibleParserError
+from ansible.plugins.inventory import BaseInventoryPlugin, to_safe_group_name
+
+from ..module_utils.client import Client
+from ..module_utils.table import TableClient
+
+
+DOCUMENTATION = r"""
+    name: servicenow.itsm.now
+    plugin_type: inventory
+    author:
+      - Manca Bizjak (@mancabizjak)
+      - Miha Dolinar (@mdolin)
+      - Tadej Borovsak (@tadeboro)
+    short_description: Inventory source for ServiceNow table records.
+    version_added: "2.10"
```

Comment:
We can either omit this (because the plugin will be part of the initial release) or set it to 1.0.0.

Reply:
This is the initial commit so I would say don't add this.